Avrip authoring (#620)
MicrosoftDocs/WSL: WSL/install-on-server.md

# Windows Server Installation Guide

> Windows Subsystem for Linux is available for installation on Windows Server 2019 (version 1709) and later. This guide will walk through the steps of enabling and installing it.

Reviewer: ```suggestion
The Windows Subsystem for Linux is available for installation on Windows Server 2019 (version 1709) and later. This guide will walk through the steps of enabling WSL on your machine.
```

2. Restart your computer when prompted. **This reboot is required** in order to ensure that WSL can initiate a trusted execution environment.

> **If you're looking for 100% system call compatibility and faster IO performance, install WSL2!**

Reviewer: We run into the same problem here as we did on the other page. I don't think we should show both install WSL2 and install WSL1 instructions on the same page. 

Here's a possible suggestion. WE can archive the old server install page (which is leaving it in the repo, but taking it off the table of contents) and add a link at the top 'IF you want to install just WSL1 then click here'. It's much much more unlikely that a user on a server SKU will have the major blockers to WSL2 (which is running VMWare or VirtualBox). Then this page will direct you to install WSL2 and will be much simpler.

Alternatively we can have a 'Install on Windows Server' and a 'Install just WSL1 on Window Server' but I like the earlier option more.

2. Restart your computer when prompted. **This reboot is required** in order to ensure that WSL can initiate a trusted execution environment.

**If you're looking for 100% system call compatibility and faster IO performance, install WSL2!**
> WSL 2 is only available in Windows 10 builds 18917 or higher

## For WSL2, enable the 'Virtual Machine Platform' optional component

> You will need to make sure that you have both the Windows Subsystem for Linux and the Virtual Machine Platform optional components installed. You can do that by running the following command in PowerShell: 

Author: Yes the WSL feature should be enabled by this point

```powershell
dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
```

> Please restart your machine to finish installing both components.

Reviewer: There should only be one restart with these instructions.
